app/controllers/GalleryController.scala

```diff
@@ -3,14 +3,17 @@ package controllers
 import javax.inject.Inject
 import controllers.headers.ProvidesHeader
 import controllers.helper.GoogleMapsHelper
+import formats.json.LabelFormat._
 import models.user._
+import models.validation._
```

Comment:
Once you've finished with everything else, make sure to remove any imports you added but no longer need